Add transient notification support in Linux (#76)
iamkroot/trakt-scrobbler: README.md

    (Windows users may require a reboot after this step)
5.  Run `pipx install trakt-scrobbler`. You will now have the `trakts` command available.  
For SOCKS proxy support, use `pipx install "trakt-scrobbler[socks]"`.

> For updatable notification support in Linux, either install your distributions package for `pygobject` or run `pipx install "trakt-scrobbler[linux_gobject]"`.

Reviewer: ```suggestion
For updatable notification support in Linux, either install your distribution's package for `pygobject` or run `pipx install "trakt-scrobbler[linux_gobject]"`.
```

> For updatable notification support in Linux, either install your distribution's package for `pygobject` or run `pipx install "trakt-scrobbler[linux_gobject]"`.

Reviewer: Just tried `poetry install -E pygobject` on a fresh environment, and it errored out. Looks like installing this with pip is not trivial at all. Had to follow steps from https://pygobject.readthedocs.io/en/latest/getting_started.html

And *then* the `-E` install worked. The install steps are getting pretty complicated now. So this should probably be an `and`. Also maybe add the link to first url above in the readme
```suggestion
For updatable notification support in Linux, either [install](https://pygobject.readthedocs.io/en/latest/getting_started.html) your distribution's package for `pygobject` and run `pipx install "trakt-scrobbler[linux_gobject]"`.
```

> For updatable notification support in Linux, either [install](https://pygobject.readthedocs.io/en/latest/getting_started.html) your distribution's package for `pygobject` and run `pipx install "trakt-scrobbler[linux_gobject]"`.

Reviewer: ```suggestion
For updatable notification support in Linux, [install](https://pygobject.readthedocs.io/en/latest/getting_started.html) your distribution's package for `pygobject` and run `pipx install "trakt-scrobbler[linux_gobject]"`.
```

6.  Run `trakts init`. You will be prompted to select the players to be monitored, and authorize the app to access the Trakt.tv API on your behalf. Follow the steps on screen to finish the process.

**For Linux:**

> To enable notification support on Linux, `pygobject` and `libnotify` needs to be installed (Reboot after installation).

Reviewer: This isn't what I meant by updating the install instructions. Since you have specified the `linux_gobject` extra install target, there is no need to install `pygobject` manually. Again, have a look at the install instructions for `pysocks` to understand where to update the README.

Also, this line isn't strictly true. `Pygobject` is only needed for richer notifications that can be updated. Otherwise, `libnotify` is enough for basic notifications. The reason `Pygobject` isn't default is that it is a very heavy library. Better to mention this too.

Author: will this be ok?
> For updatable notification support in Linux, either install your distributions package for `pygobject` or run `pipx install "trakt-scrobbler[linux_gobject]"`.

Reviewer: That looks great
